snippets/09_facebook.txt

```diff
@@ -83,6 +83,7 @@
 2a03:2880:f00c:19:face:b00c:0:3 scontent-b-sin.xx.fbcdn.net
 2a03:2880:f00d:8:face:b00c:0:1 scontent-a-lax.xx.fbcdn.net
 2a03:2880:f00d:8:face:b00c:0:1 scontent-b-lax.xx.fbcdn.net
+2a03:2880:f007:8:face:b00c:0:1 scontent-vie1-1.xx.fbcdn.net

```

Comment:
Should this line be in https://github.com/lennylxx/ipv6-hosts/pull/215/?

Reply:
emm... maybe a merge conflict? You can see I hard merged your branch. VS code may just ignored this conflict line?